source/includes/fact-mongoshrc-loading.rst

```diff
@@ -11,10 +11,10 @@ on the command line or by specifying a :ref:`.js file
 .mongoshrc.js from being loaded by using the :option:`--norc
 <mongosh --norc>` option.
 
-.. note::
+.. note:: Legacy Shell Configuration 
 
-   The legacy :binary:`~bin.mongo` shell used a configuration file
-   called .mongorc.js. If :binary:`mongosh` detects this file on
-   startup, and .mongoshrc.js is not present, :binary:`mongosh` does not
-   load the legacy .mongorc.js file, and suggests renaming .mongorc.js
-   to .mongoshrc.js.
+   The legacy :binary:`~bin.mongo` shell used a configuration file called 
+   ``.mongorc.js``. If :binary:`mongosh` detects this file on startup, and 
+   ``.mongoshrc.js`` is not present, :binary:`mongosh` does not load the legacy 
+   ``.mongorc.js file``, and suggests renaming ``.mongorc.js`` to 
```

Comment:
I don't think we want "file" to be monospaced.

```suggestion
   ``.mongorc.js`` file, and suggests renaming ``.mongorc.js`` to 
```

Reply:
Ah thanks for the catch. Fixed!